docs/content/dagster-cloud/developing-testing/setting-environment-variables-dagster-cloud-agents.mdx

```diff
@@ -22,15 +22,20 @@ To complete the steps in this guide, you'll need:
 
 - A Dagster Cloud account using [Hybrid deployment](/dagster-cloud/deployment/hybrid)
 - An existing [Hybrid agent](/dagster-cloud/deployment/agents)
-- Editor, Admin, or Organization Admin permissions in Dagster Cloud
+- **Editor**, **Admin**, or **Organization Admin** permissions in Dagster Cloud
 
 ---
 
 ## Setting environment variables for a code location
 
 <Note>
-  <strong>Editor, Admin, or Organization Admin</strong> permissions are required
-  to modify and redeploy code locations in Dagster Cloud.
+  If you're a Dagster Cloud{" "}
+  <a href="/dagster-cloud/account/managing-users">
+    <strong>Editor</strong> or <strong>Admin</strong>
+  </a>
+  , you can only set environment variables for full deployments where you're an <strong>
+    Editor
+  </strong> or <strong>Admin</strong>.
```

Comment:
do we want to mention org admin in this and the similar one?

Reply:
Yes, thanks for catching this! I accidentally pasted over the first part of this, which included the org admin bit.